lib/functions/aeminitiateupload.js

```diff
@@ -104,11 +106,27 @@ class AEMInitiateUpload extends AsyncGeneratorFunction {
             const completeUrl = new URL(initiateResponse.completeURI, folderUrl);
             for (let i = 0; i < assets.length; ++i) {
                 const transferAsset = assets[i];
-                const { minPartSize, maxPartSize, uploadURIs, uploadToken } = files[i];
+                const { minPartSize, maxPartSize, uploadURIs, mimeType, uploadToken } = files[i];
+
                 // uploadToken is opaque to us, we just need to send it back with the completeUpload
                 if (!Number.isFinite(minPartSize) || !Number.isFinite(maxPartSize) || !Array.isArray(uploadURIs) || (uploadURIs.length === 0)) {
                     throw Error(`invalid multi-part information for ${transferAsset.target.url}: ${JSON.stringify(files[i])}`);
                 }
+
+                // if the client has no mimetype specified, use the mimetype provided by AEM, if AEM
```

Comment:
Nitpick to put the `if AEM` on the second line, it reads a little bit better 
```suggestion
                // if the client has no mimetype specified, use the mimetype provided by AEM
```